released/packages/coq-math-classes/coq-math-classes.8.9.1/opam

```diff
@@ -0,0 +1,32 @@
+opam-version: "2.0"
+maintainer: "b.a.w.spitters@gmail.com"
+
+homepage: "https://github.com/coq-community/math-classes"
+dev-repo: "git+https://github.com/coq-community/math-classes.git"
+bug-reports: "https://github.com/coq-community/math-classes/issues"
+license: "Public Domain"
+
+synopsis: ""
+description: """
+A library of abstract interfaces for mathematical structures in Coq.
+"""
+
+build: [
+  [ "./configure.sh" ]
+  [ make "-j%{jobs}%" ]
+]
+install: [make "install"]
+depends: [
+  "ocaml"
+  "coq" {(>= "8.6" & < "8.11~")}
+  "coq-bignums" 
+]
+
+tags: [
+  "logpath:MathClasses"
+]
+authors: [
+  "Eelis van der Weegen"
+  "Bas Spitters"
+  "Robbert Krebbers"
+]
```

Comment:
Here is why the CI currently fails (I didn't catch this in my first read-through):
```suggestion
]

url {
  src: "https://github.com/coq-community/math-classes/archive/8.9.1.tar.gz"
  checksum: "sha256=2f35df200214be783c8cc9b959482f80169bb25526b6e19b0919a7b20bf7f7ef"
}
```